tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -170,10 +170,13 @@ void UpdateLaunchDimensions(const LaunchDimensions& launch_dims, Thunk* thunk,
 
 }  // namespace
 
-IrEmitterUnnested::IrEmitterUnnested(const HloModuleConfig& hlo_module_config,
-                                     const HloComputation* hlo_computation,
-                                     IrEmitterContext* ir_emitter_context)
-    : IrEmitter(hlo_module_config, ir_emitter_context, /*is_nested=*/false),
+IrEmitterUnnested::IrEmitterUnnested(
+    const HloModuleConfig& hlo_module_config,
+    const HloComputation* hlo_computation, IrEmitterContext* ir_emitter_context,
+    llvm_ir::LLVMTargetFeatures* llvm_target_features)
+
```

Comment:
No newline (although you don't have to change this, it'll be fixed when we import the PR).